File: MPP/MPP_Iteratia3_Java/Server/src/main/java/StartServer.java
import Repository.BiletRepo;
import Repository.ConcertRepo;
import Repository.UserRepo;

public class StartServer {
    private static int defaultPort = 55555;

    public static void main(String[] args) {
        UserRepo ur = new UserRepo();
        ConcertRepo cr = new ConcertRepo();
        BiletRepo br = new BiletRepo();

        IService service = new ServiceImplemented(ur, br, cr);
        int serverPort = 55556;
        AbstractServer server = new ObjectConcurrentServer(serverPort, service);
        try{
            server.start();
        }catch (Exception e){
            e.printStackTrace();
        }
    }
}

File: MPP/MPP_Iteratia3_C#/StartServer.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Server
{
    using Server;
    using Server.repos;

    public class StartServer
    {
        static void Main(string[] args)
        {
            UserRepo userRepo = new UserRepo();
            ConcertRepo concertRepo = new ConcertRepo();
            BiletRepo biletRepo = new BiletRepo();

            RealService service = new RealService(biletRepo, concertRepo, userRepo);
            MyConcurrentServer server = new MyConcurrentServer("127.0.0.1", 1234, service);
            Console.Write("Concurrent Server created ...  \n ... ");
            server.Start();
            Console.Write(" started ... ");
            Console.ReadLine();
        }
    }
}
